closes  #7. Migrating the Bugzilla Workshop from https://github.com/OSLC/integrating-products-with-oslc-tutorial

jadelkhoury opened the pull request:
closes  #7. Migrating the Bugzilla Workshop from https://github.com/OSLC/integrating-products-with-oslc-tutorial

jadelkhoury committed:
closes  #7.

Migrating the Bugzilla Workshop from
https://github.com/OSLC/integrating-products-with-oslc-tutorial

jadelkhoury commented:
Status: I have done a thourough check of the links. Seems quite stable now, except for the  /integrating_products_with_oslc/running_the_examples page.

What is left to do. First point is most critical:

- [x] Locate the source code git://git.eclipse.org/gitroot/lyo/org.eclipse.lyo.docs.git. https://github.com/eclipse/lyo.docs does not seem to match, if one sees for example the package names used.
- [x] Finalize and simplify the page /integrating_products_with_oslc/running_the_examples
- [x] Update the references/links to the tutorial from all over this DeveloperGuide to point to this new content.
- [x] Update Readme file at https://github.com/OSLC/integrating-products-with-oslc-tutorial to point to Developer Guide.
- [x] Update links/references at open-services.net to point Developer Guide.
- [x] Update Readme.md file at https://github.com/OSLC/integrating-products-with-oslc-tutorial to point to DeveloperGuide.

berezovskyi commented:
Are you blocked by any other issue? I suggest removing the label if not. It is to indicate that something else is blocking you.

jadelkhoury commented:
I’m blocked because I can’t find the source code. Where is the old git://git.eclipse.org/gitroot/lyo/org.eclipse.lyo.docs.git?

berezovskyi commented:
I see, sorry! https://github.com/eclipse/lyo.docs



--
/Andrew
(from phone)



On Wed, May 22, 2019 at 7:48 AM +0200, "Jad El-khoury" <notifications@github.com<mailto:notifications@github.com>> wrote:


I’m blocked because I can’t find the source code. Where is the old git://git.eclipse.org/gitroot/lyo/org.eclipse.lyo.docs.git?

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://github.com/OSLC/developing-oslc-applications/pull/10?email_source=notifications&email_token=AAAPZXRXQ3V7ES5H2KTOD7DPWTNB3A5CNFSM4HOPFV42YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODV5652I#issuecomment-494661353>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAAPZXQI4722K2HIFBGLFTTPWTNB3ANCNFSM4HOPFV4Q>.

jadelkhoury commented:
> I see, sorry! https://github.com/eclipse/lyo.docs

That's what I first htought, but something tells me it is not hte same code. I will look into Lab6 in details and see if it is good enough.

@berezovskyi what version of Lyo do you think we should attempt to specify in the pom files? 2.4.0 sounds too risky. What is the most stable and earlier versions we have released?

I just want to make this tutorial public for now. I don't care if it follows the latest development - yet.

berezovskyi commented:
2.3.0 should be safe. It was used by people for almost a year without any major bug reports.

jadelkhoury committed:
closes  #7.

Migrating the Bugzilla Workshop from
https://github.com/OSLC/integrating-products-with-oslc-tutorial

jadelkhoury commented:
I'm pretty happy with the final result. So, if anyone wants to do a final walkthrough, I am happy to merge.

jadelkhoury commented:
I have forgotten, do I need to compile the html pages locally, or is that taken care of at the server end?

berezovskyi commented:
https://github.com/OSLC/oslc-site-hugo/blob/master/content/resources/eclipse-lyo.md

berezovskyi commented:
what do you think? the URL will remain the same for search engine reasons.

berezovskyi commented:
you need to run the deploy script in the root folder

berezovskyi commented:
just deployed

jadelkhoury commented:
> 
> 
> what do you think? the URL will remain the same for search engine reasons.

@berezovskyi! You are on the wrong issue.

berezovskyi commented:
oh yes I am

berezovskyi commented:
> I have forgotten, do I need to compile the html pages locally, or is that taken care of at the server end?

GH will take care of everything

berezovskyi commented on _harp/_data.json:
```diff
@@ -14,8 +14,8 @@
   },
-  "sample-applications-and-code": {
-    "title": "Sample applications and code"
+  "tutorials-and-sample-applications": {
```
aghh, this will break links and kick us off the (already low) positions in Google Search...

berezovskyi commented:
@jadelkhoury I forgo, you actually have to re-compile Harp site first and then Github will publish the HTML automatically.

jadelkhoury committed:
closes  #7.

Migrating the Bugzilla Workshop from
https://github.com/OSLC/integrating-products-with-oslc-tutorial

restoring the page for the url "sample-applications-and-code"

Also updating the links to the oslc-primer on open-services.net.

jadelkhoury committed:
closes  #7.

Migrating the Bugzilla Workshop from
https://github.com/OSLC/integrating-products-with-oslc-tutorial

generated html pages based on latest md files.

jadelkhoury commented:
@berezovskyi 
I fixed the borken link to "sample-applications-and-code". Plus, regenerated the html. 
Many files were generated it seems, but I think it is safe.

Anything odd you can see? Otherwise, ready to merge.

berezovskyi commented:
Look good, please go ahead @jadelkhoury
